docs/codeql/codeql-workshops/cpp/codeql-dataflow-sql-injection/Makefile

```diff
@@ -0,0 +1,26 @@
+add-user: add-user.c
```

Comment:
I wonder if, within each workshop directory, it's worth separating the target source code (`src`) and the CodeQL queries written against that code into separate directories. That will make it clearer that the ad-hoc build scripts are to do with the target code, not CodeQL itself.